swap_meet/item.py

```diff
@@ -1,2 +1,24 @@
 class Item:
-    pass
+
+    def __init__(self, category=None, condition = 0):
+        self.category = category if category is not None else ""
+        self.condition = condition + 5
+
+    def __str__(self): 
+        return f'Hello World!'
+    
+    def condition_description(self): 
+        if self.condition == 0: 
+            return("trash") 
```

Comment:
more often than not you'll see the return statement of a single value without the parentheses

```suggestion
            return "trash"
```